custom/panel_templates/Default/template.php

```diff
@@ -412,6 +412,15 @@ public function onPageLoad() {
 							');
 
                         }
+                        
+                        $this->addJSScript('
+							var elems = Array.prototype.slice.call(document.querySelectorAll(\'.js-switch\'));
+
+							elems.forEach(function(html) {
+							  var switchery = new Switchery(html, {color: \'#23923d\', secondaryColor: \'#e56464\'});
```

Comment:
switchery has been removed for a while. i think this is just leftover?

Reply:
Yea it is